src/mask.c

```diff
@@ -1437,9 +1442,9 @@ static void truncate_mask(mask_cpu_context *cpu_mask_ctx, int range_idx)
 
 	if (range_idx < mask_max_skip_loc && mask_max_skip_loc != -1) {
 		if (john_main_process)
-			fprintf(stderr, "Format internal ranges (first %d range positions)"
-			        " cannot be truncated!\n"
-			        "Use a larger min-length, or non-gpu format.\n",
+			fprintf(stderr,
+			        "Error: Format's internal mask ranges (first %d positions) cannot be truncated!\n"
+			        "       Increase minimum length, or use the CPU format.\n",
```

Comment:
This implies we don't have CPU formats with internal mask, but I think ideally we should add that to the LM format. Maybe make the ", or use the CPU format" portion conditional on "opencl" or "ztex" in the format name?

Reply:
Very few formats have this limitation, only Sayantan's "worst" that I couldn't fix. I changed the wording to more neutral